quality/fca/cypress/plugins/log-plugin.ts

```diff
@@ -1,6 +1,8 @@
 import { exec } from 'child_process';
 import { promisify } from 'util';
 
+/*global Record*/
```

Comment:
question: I do not think this is the way. Record is available in the back folder and we do not need to add a coment on every file using it. We should find a cleaner way to use this.